tests/Finder/TestFrameworkFinderTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Tests\Finder;
+
+use Infection\Finder\Exception\FinderException;
+use Infection\Finder\TestFrameworkFinder;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Filesystem\Filesystem;
+
+class TestFrameworkFinderTest extends TestCase
```

Comment:
Please, use setUp and tearDown methods to create and remove temp folders:

https://github.com/infection/infection/blob/12bbf5ab87d840c8b64ec973de5e1506d27087b5/tests/TestFramework/PhpUnit/Config/Builder/InitialConfigBuilderTest.php#L49-L64